thoth_user_api/swagger.yaml

```diff
@@ -606,3 +612,43 @@ definitions:
           type: string
           description: Value of the supplied environment variable.
           example: '1'
+  Metadata:
+    type: object
+    additionalProperties: false
+    properties:
+      apiVersion:
+        type: string
+        description: >
+          BuildLog api version.
+        example: >
+          "buildlog.thoth-station.ninja/v1"
```

Comment:
we need a little bit of doc on this, pls refer to the OpenShift API schema and see if they got some useful comments on their usage of API versions.

Reply:
/apis/build.openshift.io/v1/namespaces/$NAMESPACE/builds HTTP/1.1 this is how it gets added, il update this